### PR TITLE
readme updated with table of contents

### PR DESCRIPTION
**Pull request by AndiRenFC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
-- ClariSpend: provides daily bill analysis and 7-day trend data by accounts and services. We also detect aberrations in your AWS usage and through our comprehensive reporting, you will be able to understand and control your AWS utilization.
+Our AI-powered product discovers unnecessary AWS cloud compute resources and eliminates them - an otherwise manual and time intensive process that is an inefficient use of human attention. We free up engineering teams to focus on what matters most: building stuff. 
 
-- Optima: automates instance downsizing, buying and selling of reserved instances, manages your savings plans, and shuts down unused instances. We provide real-time notifications for saving opportunities as well as easy-to-access action buttons directly within Slack. In addition, you will be able to manage inventories and generate visualizations and reports using our web portal.
+## Components
+
+### ClariSpend
+Provides daily bill analysis and 7-day trend data by accounts and services. We also detect aberrations in your AWS usage and through our comprehensive reporting, you will be able to understand and control your AWS utilization.
+
+### Optima
```
Comment (ShanForma) [inline README.md:22]: https://forma-cloud.slack.com/archives/C047XB0TJGK/p1689382202788809

**Review by ShanForma — COMMENTED** [review]
Also change line 74 please. Could we proofread the whole doc?

File: README.md (hunk 1 above)
Comment (AndiRenFC) [reply README.md]: I added all four components based on the thread, please read through them and see if they are accurate.

File: README.md [hunk 2]
```diff
@@ -1,35 +1,72 @@
...
+
+### ClariSpend
+Provides daily billing, utilization, and savings reports with weekly trend data, grouped by accounts and services. ClariSpend also detects and highlights irregularities in your AWS usage to help you detect anomalies and react early. Through our comprehensive reporting, you will be able to understand and stay up to date on your AWS usage and how much we are saving for you.
+
+### Saving Bot
+Intelligently shuts down and rightsizes EC2 instances with human supervision. Saving bot monitors your EC2 instances and notifies you of under-utilized ones that eat away your cloud costs. The Saving bot can be configured to automatically shut down some instances, while only acting after human confirmation on instances that are risky to shut down. Rightsizing adjusts the instance type to match the usage, without harming or altering the instance's content.
```
Comment (ShanForma) [inline README.md:29]: Use "Saving Bot" for consistency.

File: README.md [hunk 3]
```diff
@@ -1,35 +1,72 @@
...
+
+### Infra Copilot (Coming soon)
+A chatbot enabled by generative AI that offers capabilities beyond those of ChatGPT. With our custom-tuned AI models, as well as access to your data, Copilot can:
+   1. Answer questions with text and visualizations specific to your cloud usage. You don't need to extract, process, and upload your own data as when dealing with ChatGPT; all you need to do is ask Copilot.
```
Comment (ShanForma) [inline README.md:33]: Remove the section of "Copilot can" before we release it into production.

**Review by ShanForma — APPROVED** [review]

**Pull request merged**